getting-started-react-cra5/src/App.js

```diff
@@ -1,21 +1,32 @@
 import "./App.css";
-import { Metaplex } from "@metaplex-foundation/js";
+import { Metaplex, walletAdapterIdentity } from "@metaplex-foundation/js";
 import { clusterApiUrl, Connection, PublicKey } from "@solana/web3.js";
+import { useWallet } from '@solana/wallet-adapter-react';
 import { useState } from "react";
 
-const connection = new Connection(clusterApiUrl("devnet"));
-const mx = Metaplex.make(connection);
 
 function App() {
   const [address, setAddress] = useState(
     "3ijFZcJKmp1EnDbbuaumWYvEFbztx9NRupwTXTchK9bP"
   );
-  const [nft, setNft] = useState(null);
 
-  const fetchNft = async () => {
-    const asset = await mx.nfts().findByMint({ mintAddress: new PublicKey(address) });
+  const [candy, setCandy] = useState(null);
 
-    setNft(asset);
+  const connection = new Connection(clusterApiUrl("devnet"));
+  const wallet = useWallet();
+  const mx = Metaplex.make(connection);
+  mx.use(walletAdapterIdentity(wallet));
+  const collectionUpdateAuthority = new PublicKey("9eYgKf44KuPocpSpC3QtLqkjtMjHTbUP4KgrC7mYz4GM");
+  
+  const mintNft = async () => {
```

Comment:
Any reason this "fetch NFT" operation was turned into a "mint NFT" operation?

Reply:
looks like I commited a in progress version before i decided to use the connect-wallet example.
reverted.